### PR TITLE
Triggers: Rerencing,Enable,Disable

### PR DESCRIPTION
**Pull request by jalissonmello**
This PR contains changes to allow use Rerencing clause, Disable Or Enable Trigger.

Suggestions are welcome

### CODE REVIEW

File: src/test/java/cz/startnet/utils/pgdiff/PgDiffTest.java [hunk 1]
```diff
@@ -312,7 +316,7 @@ public PgDiffTest(final String fileNameTemplate,
      *
      * @throws FileNotFoundException Thrown if expected diff file was not found.
      * @throws IOException           Thrown if problem occurred while reading
-     *                               expected diff.
+     *                               expected diff.'1
```
Comment (valoricDe) [inline src/test/java/cz/startnet/utils/pgdiff/PgDiffTest.java:319]: was  this accidentaly?

**Comment by valoricDe** [comment]
Hmm when I would merge that I would revert the sequence code already merged into develop right?
Appearantly it would atleast in my fork: https://github.com/valorize/apgdiff/pull/4

File: src/test/java/cz/startnet/utils/pgdiff/PgDiffTest.java (hunk 1 above)
Comment (jalissonmello) [reply src/test/java/cz/startnet/utils/pgdiff/PgDiffTest.java:319]: Yes

**Comment by jalissonmello** [comment]
Since this PR is removing code, I created a new one with all my all changes and others that is waiting on develop

**Comment by valoricDe** [comment]
Closing, coz merged in #245

**Pull request closed without merging**